catalyst-gateway/bin/src/db/index/schema/mod.rs

```diff
@@ -219,21 +222,12 @@ pub(crate) async fn create_schema(
         .await
         .context("Creating Namespace")?;
 
-    let mut failed = false;
-
-    for (schema, schema_name) in SCHEMAS {
-        match session.prepare(*schema).await {
-            Ok(stmt) => {
-                if let Err(err) = session.execute_unpaged(&stmt, ()).await {
-                    failed = true;
-                    error!(schema=schema_name, error=%err, "Failed to Execute Create Schema Query");
-                };
-            },
-            Err(err) => {
-                failed = true;
-                error!(schema=schema_name, error=%err, "Failed to Prepare Create Schema Query");
-            },
-        }
+    let failed = false;
+
+    thread::sleep(Duration::from_secs(KEYSPACE_LATENCY));
+
+    for (schema, _schema_name) in SCHEMAS {
+        session.query_unpaged((*schema).to_string(), &[]).await?;
     }
```

Comment:
After the schema creations are run, we need to poll them to ensure the tables are created, as they also are not immediate.
See: https://docs.aws.amazon.com/keyspaces/latest/devguide/tables-create.html for details.